src/python/doc/perslay.md

```diff
@@ -0,0 +1,135 @@
+    | **mean_const**         | Boolean specifying if **mean_init** is initialized with a value (True) or randomly with tensorflow (False). |
+    | **variance_init**      | Initializer of the bandwidths of the rational structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    | **variance_const**     | Boolean specifying if **variance_init** is initialized with a value (True) or randomly with tensorflow (False). |
+    | **alpha_init**         | Initializer of the exponents of the rational structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    | **alpha_const**        | Boolean specifying if **alpha_init** is initialized with a value (True) or randomly with tensorflow (False). |
+    | **fc_layers**          | Sequence of fully-connected operations to be applied after the permutation invariant operation. It is a list of tuples of the form (*dim*, *pro*, *dropout*). Each tuple defines a fully-connected operation, with dimension *dim* (integer) and processing *pro* (string, e.g. "bdr" ---> batch-norm, dropout, relu). If there is a "d" in string, i.e. dropout, the dropout value can be specified with *dro* (float, default 0.9). Example: [(150,"br"), (75, "bd", 0.85)].|
+
+  * if **layer** is "rh":
+
+    | **name** | **description** |
+    | --- | --- |
+    | **num_elements**      | Number of rational hat structure elements that will be evaluated on the persistence diagram points. |
+    | **mean_init**         | Initializer of the means of the rational hat structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(0., 1.). |
+    | **mean_const**        | Boolean specifying if **mean_init** is initialized with a value (True) or randomly with tensorflow (False). |
+    | **r_init**            | Initializer of the threshold of the rational hat structure elements. It can be either a numpy array of values, or a random initializer from tensorflow, such as tensorflow.random_uniform_initializer(3., 3.). | 
+    | **r_const**           | Boolean specifying if **r_init** is initialized with a value (True) or randomly with tensorflow (False). |
+    | **q**                 | Norm parameter. |
+    | **fc_layers**         | Sequence of fully-connected operations to be applied after the permutation invariant operation. It is a list of tuples of the form (*dim*, *pro*, *dropout*). Each tuple defines a fully-connected operation, with dimension *dim* (integer) and processing *pro* (string, e.g. "bdr" ---> batch-norm, dropout, relu). If there is a "d" in string, i.e. dropout, the dropout value can be specified with *dro* (float, default 0.9). Example: [(150,"br"), (75, "bd", 0.85)].|
```

Comment:
Overall, I think we should redefine what a layer is, to avoid having unrelated variables at the same level ; in my mind this should rather have a tree structure, like
`perslay_param` might decompose in `permutation_operator`, `weight_function` and `vectorization_function`, each of them being independently instantiated.
As such, maybe using a Class would be better.
It might also help to have a clearer doc.